sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -739,7 +738,8 @@ static void set_codec_init_func(const struct snd_soc_acpi_link_adr *link,
 static int create_sdw_codec_dai(struct device *dev,
 				struct snd_soc_dai_link *dai_links, int *id,
 				struct snd_soc_dai_link_component *cpus,
-				const struct snd_soc_acpi_link_adr *link)
+				const struct snd_soc_acpi_link_adr *link,
```

Comment:
@RanderWang the name of this func is misleading. We're not really creating a dai here, are we? Should be renamed o dai_link, I think

Reply:
yes,  Pierre also recommended "create_sdw_dailink" and I fixed it.

Reply:
@RanderWang it's still create_sdw_codec_dai() in this latest version